pages/common/dvtm.md

```diff
@@ -0,0 +1,32 @@
+# dvtm
+
+> Dynamic virtual terminal manager.
+
+- Show manual page:
+
+`Ctrl+g ?`
+
+- Create a new shell window:
+
+`Ctrl+g c`
+
+- Focus the n-th window:
+
+`Ctrl+g n`
+
+- Close focused window:
+
+`Ctrl+g x x`
+`Ctrl+d (for shell only)`
+
+- Toggle between defined layouts:
+
+`Ctrl+g Space`
+
+- Toggle dvtm mouse grabbing:
```

Comment:
No need to mention dvtm here and the last example. It is understood.

Reply:
Ok. I just took it from the manual.

Reply:
We always try to avoid the language in the manual. Otherwise it will simply be a copy-paste job and the user can anyways lookup the manual. There will be no added value for this page. :)